recipes/sip/bld.bat

```diff
@@ -0,0 +1,8 @@
+%PYTHON% configure.py --sysroot=%PREFIX% --bindir=%LIBRARY_BIN%
```

Comment:
Sorry, just thinking about one more thing here. Do you think we should add ` --incdir` too? Just wondering since we had to specify the binary location.